README.md

```diff
@@ -45,7 +45,7 @@ Project highlights:
 ## Requirements ##
 
 `Logisim-evolution` is a Java application; therefore, it can run on any operating system supporting the Java runtime enviroment.
-It requires [Java 16 (or newer)](https://www.oracle.com/java/technologies/javase-downloads.html).
+It requires [Java 21 (or newer)](https://www.oracle.com/java/technologies/javase-downloads.html).
```

Comment:
Maybe, we could consider changing the download link to the [Eclipse Temurin project](https://adoptium.net/de/temurin/releases/), as it provides OpenJDK builds for more platforms.

Reply:
Done.